components/http_kernel.rst

```diff
@@ -742,6 +742,11 @@ look like this::
 
 .. _http-kernel-resource-locator:
```

Comment:
this must be moved above the next Headline `Locating Resources` again

Reply:
Done while merging. Thanks.